.eslintrc.json

```diff
@@ -16,7 +16,6 @@
     "camelcase": 2,
     "curly": 2,
     "eqeqeq": [2, "smart"],
-    "func-style": [2, "expression"],
```

Comment:
I removed this because it violates the Airbnb React Style Guide.
https://github.com/airbnb/javascript/blob/master/react/README.md#class-vs-reactcreateclass-vs-stateless
> And if you don't have state or refs, prefer normal functions (not arrow functions) over classes:

```javascript
// bad
class Listing extends React.Component {
  render() {
    return <div>{this.props.hello}</div>;
  }
}

// bad (relying on function name inference is discouraged)
const Listing = ({ hello }) => (
  <div>{hello}</div>
);

// good
function Listing({ hello }) {
  return <div>{hello}</div>;
}
```

Reply:
What's "function name inference" and why does AirBnB consider it bad?

Reply:
I had no idea, so I dug up some resources!
* https://github.com/airbnb/javascript/issues/794
* https://stackoverflow.com/questions/37288950/why-does-the-airbnb-style-guide-say-that-relying-on-function-name-inference-is-d

Sooo anonymous functions don't have a name, so it's hard to debug when we see them show up in the call stack.
`const x = function () { }`

It's a lot better when the function is named, even if it's been assigned to a variable thereafter.
`const shortName = function longReallyDescriptiveName() { }`
(I hadn't actually seen this before... I like the idea!)

Thankfully, sometimes the function name is _inferred_ by transpilers/browsers where `const foo = function() { }` might be understood as `const foo = function foo(){ }`, but Airbnb style guide says that's bad bc explicit is better than implicit.

....after researching this, I personally don't care whether we prefer `function ComponentName{ }` or `const ComponentName = function() { }` for stateless components, but was just trying to be a good "role model" by doing what the style guide said. ¯\_(ツ)_/¯

@delventhalz Do you have a preference?

Reply:
I've always considered function declarations to lead to errors and confusion, and naming functions you're assigning to a variable to be redundant and messy (or confusing if you give them different names). I also don't think I've ever had an issue debugging them. So I always use anonymous function expressions, and that seems to be the norm, but I'm down with AirBnB if we want to go that route. I was just curious what their reasoning was.